tiles/config.toml.template

```diff
@@ -9,13 +9,13 @@ dbconn = "postgresql://cosmogony:cosmogony@postgres/cosmogony"
 predefined = "web_mercator"
 
 [[tileset]]
-name = "cosmogony"
+name = "$TILESET_NAME"
 
 [[tileset.layer]]
 name = "vector-zones"
 datasource = "cosmogony"
 # Select all attributes of table:
-table_name = "zones"
+table_name = "$TABLE_SPACE.zones"
```

Comment:
The table name needs to be updated in the custom queries below. Actually this `table_name` may not be required.